lib/framework/data_context.dart

```diff
@@ -365,6 +367,16 @@ class NativeInvokable extends ActionInvokable {
         final scope = ScreenController().getScopeManager(buildContext);
         callNativeMethod(buildContext, scope, inputs);
       },
+      ActionType.showBottomModal.name: (inputs) => ScreenController()
```

Comment:
This is the old way of registering it. Please put them in the super class ActionInvokable